src/constants.rs

```diff
@@ -47,7 +59,7 @@ mod tests {
         let val = br#"form-data; name="my_field""#;
         let name = ContentDispositionAttr::Name.extract_from(val);
         let filename = ContentDispositionAttr::FileName.extract_from(val);
-        assert_eq!(name.unwrap(), b"my_field");
+        assert_eq!(name.unwrap(), "my_field".as_bytes());
```

Comment:
Why these changes?

Reply:
I think it was because I was using find/replace to convert from strings back to bytes, and at least one of the tests requires as_bytes. I can go about undoing these changes if you want, but I kinda like the consistency.